### ISSUE TITLE
Cannot install tensorflow-gpu==1.15.0

### ISSUE THREAD

**Issue by dvogureckiy99** [tensorflow/tensorflow#55582]
**System information**
- OS Platform and Distribution: Linux Ubuntu 20.04
- TensorFlow version: tensorflow-gpu==1.15.0
- Python version: 3.17.13
- Installed using virtualenv? pip? conda?: pip
- CUDA/cuDNN version: 
- GPU model and memory: 

Describe the problem
I get the following error when trying to install tensorflow-gpu in version  1.15.0:
```
Using pip 22.0.4 from /home/dmitriy/.local/lib/python3.8/site-packages/pip (python 3.8)
ERROR: Could not find a version that satisfies the requirement tensorflow-gpu==1.15.0 (from versions: 2.2.0, 2.2.1, 2.2.2, 2.2.3, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.3.4, 2.4.0, 2.4.1, 2.4.2, 2.4.3, 2.4.4, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.6.0, 2.6.1, 2.6.2, 2.6.3, 2.7.0rc0, 2.7.0rc1, 2.7.0, 2.7.1, 2.8.0rc0, 2.8.0rc1, 2.8.0)
ERROR: No matching distribution found for tensorflow-gpu==1.15.0
```

**Comment by mihaimaruseac** [comment]
TF 1.15 is too old. When it was released there was no python 3.8.

This is not a TF issue.

**Comment by dvogureckiy99** [comment]
Okey, but there are TF-gpu required. How can I choose another TF version ?
